src/plugins/visualize/public/application/components/visualize_editor_common.tsx

```diff
@@ -126,6 +131,7 @@ export const VisualizeEditorCommon = ({
         />
       )}
       {visInstance?.vis?.type?.stage === 'experimental' && <ExperimentalVisInfo />}
+      {!hasHeatmapLegacyhartsEnabled && isSplitChart && <SplitChartWarning />}
```

Comment:
ℹ️ Displays a notification (visible only from the editor) for the heatmap charts with the new implementation and the split chart bucket set to column or row. This appears only for already existing charts (I dont allow the split chart for new charts).
![image](https://user-images.githubusercontent.com/17003240/141961995-0bd0b40e-2713-4533-a65f-b9aa704659cc.png)